src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -60,17 +61,18 @@ const getSwapperManager = (): SwapperManager => {
   const adapterManager = getChainAdapters()
   const web3 = getWeb3Instance()
 
-  // TODO: Uncomment when we are ready for a Thorchain swapper
-  // ;(async () => {
-  //   const midgardUrl = getConfig().REACT_APP_MIDGARD_URL
-  //   const thorSwapper = new ThorchainSwapper({
-  //     midgardUrl,
-  //     adapterManager,
-  //     web3,
-  //   })
-  //   await thorSwapper.initialize()
-  //   swapperManager.addSwapper(thorSwapper)
-  // })()
+  if (getConfig().REACT_APP_FEATURE_THOR) {
+    ;(async () => {
+      const midgardUrl = getConfig().REACT_APP_MIDGARD_URL
+      const thorSwapper = new ThorchainSwapper({
+        midgardUrl,
+        adapterManager,
+        web3,
+      })
+      await thorSwapper.initialize()
+      _swapperManager.addSwapper(thorSwapper)
+    })()
```

Comment:
We're missing an `await` on this async function:

```suggestion
    await (async () => {
      const midgardUrl = getConfig().REACT_APP_MIDGARD_URL
      const thorSwapper = new ThorchainSwapper({
        midgardUrl,
        adapterManager,
        web3,
      })
      await thorSwapper.initialize()
      _swapperManager.addSwapper(thorSwapper)
    })()
```